### PR TITLE
doc: add conda setting

### LINKED ISSUE #1867: Document the "python.condaPath" setting
Will be a part of the 2018.8.1 release. Basically lets people manually specify where the `conda` command lives.

### PR DESCRIPTION
**Pull request by lorderikir**
Added documentation in `settings-reference.md` on the new `python.condaPath` setting. 

This will resolve #1867

### CODE REVIEW

**Comment by kraigb** [comment]
FYI, I'm holding off on this one until the next release is out, setting don't merge accordingly.

**Comment by kraigb** [comment]
@lorderikir Just to say, I really appreciate you taking care of some of the open issues; it's typically something I spend time on with each release, and given that I have some other high-priority work, your timing is excellent. 👍

File: docs/python/settings-reference.md [hunk 1]
```diff
@@ -18,6 +18,7 @@ Refer to [User and workspace settings](/docs/getstarted/settings.md) to find our
 
 | Setting | Default | Description |
 | --- | --- | --- |
+| python.condaPath | `"conda"` | Path to the conda command, or the path to a folder containg the Conda interpreter. |
```
Comment (brettcannon) [inline docs/python/settings-reference.md]: "Path to the `conda` executable."

Basically the folder isn't a valid option and there's a typo for "containg".

**Comment by brettcannon** [comment]
Release just went out, but there's a needed tweak to the wording before this can go live.

**Comment by kraigb** [comment]
Thanks; I fixed that line and merged.

**Comment by gaziqbal** [comment]
More context would be helpful. Why would I want to specify the path to my conda executable? I've been using conda environments without this setting.

**Comment by kraigb** [comment]
@gaziqbal Could you open an issue for this on https://github.com/Microsoft/vscode-docs/issues--easier to track that way as opposed to a comment in an already-merged PR.

**Pull request merged**